composer.json

```diff
@@ -222,47 +175,46 @@
         "patchLevel": {
             "drupal/core": "-p2"
         },
-        "installer-types": ["library"],
+        "drupal-scaffold": {
+            "locations": {
+                "web-root": "web/"
+            }
+        },
         "installer-paths": {
-            "web/core": [
-                "type:drupal-core"
-            ],
+            "web/core": ["type:drupal-core"],
             "web/libraries/{$name}": [
                 "type:drupal-library",
                 "harvesthq/chosen"
             ],
-            "web/modules/custom": [
-                "dof-dss/nidirect-site-modules"
-            ],
-            "web/modules/origins": [
-                "dof-dss/nicsdru_origins_modules"
-            ],
-            "web/modules/migrate/{$name}": [
-                "dof-dss/nidirect-migrations"
-            ],
-            "web/profiles/custom": [
-                "dof-dss/nidirect-d8-test-install-profile"
-            ],
-            "web/modules/contrib/{$name}": [
-                "type:drupal-module"
-            ],
-            "web/profiles/contrib/{$name}": [
-                "type:drupal-profile"
-            ],
-            "web/themes/contrib/{$name}": [
-                "type:drupal-theme"
-            ],
-            "web/themes/custom/{$name}": [
-                "type:drupal-custom-theme"
-            ],
-            "drush/Commands/{$name}": [
-                "type:drupal-drush"
-            ]
+            "web/modules/custom": ["dof-dss/nidirect-site-modules"],
+            "web/modules/origins": ["dof-dss/nicsdru_origins_modules"],
+            "web/modules/migrate/{$name}": ["dof-dss/nidirect-migrations"],
+            "web/modules/contrib/{$name}": ["type:drupal-module"],
+            "web/modules/custom/{$name}": ["type:drupal-custom-module"],
+            "web/themes/custom/{$name}": ["type:drupal-custom-theme"],
+            "web/themes/contrib/{$name}": ["type:drupal-theme"],
```

Comment:
I can't quite see how the theme repos will be pulled in, as they have been deleted from the top of this file  and I can't now see a link to dof-dss/nicsdru_nidirect_theme.git or git@github.com:dof-dss/nicsdru_origins_theme.git anywhere in this file ?

Reply:
The themes are included as packages on lines 39 and 41

Reply:
Ah yes I see, the diff was confusing me !